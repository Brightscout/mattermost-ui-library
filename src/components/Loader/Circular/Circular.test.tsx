import React from 'react';
import {shallow} from 'enzyme';

import CircularLoader from '.';

describe('Circular Loader', () => {
    it('Should render correctly', () => {
        const component = shallow(<CircularLoader/>);
        expect(component).toMatchSnapshot();
    });
});
