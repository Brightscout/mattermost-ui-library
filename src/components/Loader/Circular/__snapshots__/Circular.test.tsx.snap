// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Circular Loader Should render correctly 1`] = `
<div
  className="loader-container d-flex align-items-center justify-content-center"
>
  <div
    className="loader-container__spinner"
  />
</div>
`;
